.github/workflows/update-apis.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  update-apis:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm install
+      - run: npm run update-disclaimers
+      - run: npm run submit-prs
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
What populates these secrets?

Reply:
`GITHUB_TOKEN` is a secret that's automatically available for all GitHub Actions, and has a basic set of permissions on the repository.  More on that here:
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

`YOSHI_CODE_BOT_TOKEN` - I'll share details offline.